backend/aspnet/MShare_ASP/API/Request/DeleteSpending.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+namespace MShare_ASP.API.Request
+{
+    /// <summary>Facing Data for a specific spending</summary>
+    public class DeleteSpending
```

Comment:
Szükségünk van ilyen sok adatra egy költés törléséhez? Nem elég csak az Id? (ha elég, akkor az is meggondolható, hogy nincs szükségünk erre az osztályra, elég egy sima long

Reply:
Módosítottam, hogy csak Id-t kapjon! Az elképzelés amúgy az volt, hogy szükség lesz a UserData-ra is az authentikáció miatt, de utólag láttam a kódot, hogy azt másképp is meg lehet oldani, és végül amellett maradtam.